AppController/djinn.rb

```diff
@@ -5743,6 +5743,7 @@ def try_to_scale_down(app_name, delta_appservers)
           @last_decision[app_name] = Time.now.to_i
           Djinn.log_info("Removing an AppServer for #{app_name} #{location}.")
           num_to_remove -= 1
+          examine_scaling_down_instances(app_name, node_ip)
```

Comment:
I see what you mean now, and I do need to put some more checks around it to make sure it does not terminate instances which weren't autoscaled.